myMigrationScripts/migratePokemonLEvelAndMoves.js

```diff
@@ -0,0 +1,70 @@
+const connectDB = require('../config/db');
+const Pokemon = require('../models/Pokemons');
+const fetch = require('node-fetch');
+
+connectDB();
+
+// Get random number between minand max
+const getRandomBetween = (min, max) => {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+// Get random moves
+const getRandomMoves = (moves, times) => {
+  // if there's less than 2 moves, just return them
+  if (moves.length <= times) return moves;
+  const result = [];
+  let randomIndex;
+  while (times > 0) {
+    randomIndex = getRandomBetween(0, moves.length - 1);
+    result.push(moves[randomIndex]);
+    moves.splice(randomIndex, 1);
+    times -= 1;
+  }
+  return result;
+};
+
+const updatePokeLevelAndMoves = async () => {
+  try {
+    //get all pokemons from db
+    const allPokemons = await Pokemon.find({});
+
+    // filter pokemons to those without level or moves
+    const pokemonsToUpdate = allPokemons.filter(
+      pokemon => !pokemon.level || !pokemon.moves || pokemon.moves.length === 0
+    );
+
+    let res;
+    let moves;
+    let data;
```

Comment:
there's no need to declare these variables here with `let` -> you can declare them as `const`s inside `forEach`